app/views/organizations/show.html.erb

```diff
@@ -59,13 +56,12 @@
   <%= link_to "Edit", edit_organization_path(@organization), class: "btn btn-primary" %>
 <% end %>
 
-<% if user_signed_in? %>
-  <% if !(@claimed && @claimed.status == true)  %>
-    <% if @organization.user == nil %>
-      <%= link_to "Claim this business", new_organization_claim_request_path(@organization), class: "btn btn-primary"%>
-    <% end %>
+<% if !(@claimed && @claimed.status == true) || @organization.user == nil  %>
```

Comment:
Try to wrap this in a helper method, it's hard to understand what's happening here